lib/ClangImporter/ClangAdapter.cpp

```diff
@@ -563,6 +563,9 @@ OptionalTypeKind importer::translateNullability(clang::NullabilityKind kind) {
   case clang::NullabilityKind::Unspecified:
     return OptionalTypeKind::OTK_ImplicitlyUnwrappedOptional;
   }
+
+  // Work around MSVC warning: not all control paths return a value
+  llvm_unreachable("All switch cases are covered");
```

Comment:
Well, by the end of my flurry of PRs, there shouldn't be any left :) The habit thing still remains